mrmustard/lab_dev/transformations/transformations.py

```diff
@@ -384,3 +384,64 @@ def representation(self) -> Bargmann:
         n_modes = len(self.modes)
         eta = list(reshape_params(n_modes, eta=self.transmissivity.value))[0]
         return Bargmann(*triples.attenuator_Abc(eta))
+
+
+class Fockdamping(Channel):
```

Comment:
Probably the best thing here is to define it as an `Operation` instead, because it's a bit of a waste to define it as a channel (it only has one Kraus operator).